tests/test_check_cloudformation.py

```diff
@@ -0,0 +1,73 @@
+from unittest.mock import patch
+
+import pytest
+from click.testing import CliRunner
+
+import commands
+from commands.check_cloudformation import check_cloudformation as check_cloudformation_command, valid_checks
+
+
+class TestCheckCloudformationCommand:
+    @pytest.fixture
```

Comment:
I think @ejayesh has created a conftest file, if you wanted to separate the fixture out into that

Reply:
Yes, pull it in and start using it.